testkit/src/main/scala/org/bitcoins/testkit/node/fixture/NodeConnectedWithBitcoind.scala

```diff
@@ -0,0 +1,10 @@
+package org.bitcoins.testkit.node.fixture
+
+import org.bitcoins.node.{NeutrinoNode, Node, SpvNode}
+import org.bitcoins.rpc.client.common.BitcoindRpcClient
+
+/** Gives us a fixture that has a SPV node connected with the bitcoind instance */
+case class NodeConnectedWithBitcoind(node: Node, bitcoind: BitcoindRpcClient) {
```

Comment:
We should create an ADT here as well, same idea as the comment I left in `NodeUnitTest.scala`